activerecord/test/models/hamster.rb

```diff
@@ -0,0 +1,4 @@
+class Hamster < ActiveRecord::Base
```

Comment:
Actually tried using an existing model but when I changed it to validate presence other tests would fail. Hadn't though of this approach, will change.